apps/docs/content/getting-started/examples.mdx

```diff
@@ -6,10 +6,11 @@ date: 10/11/2023
 order: 3
 ---
 
-Here's a [very simple sandbox](https://codesandbox.io/s/tldraw-example-canary-2lrzmy) showing the tldraw component.
+Below is a [simple sandbox](https://codesandbox.io/s/tldraw-example-canary-2lrzmy) showing the tldraw component.
```

Comment:
Let's cut this paragraph - I think it's self-evident that it's the sandbox is below (which should go away anyway soon)